src/util.h

```diff
@@ -589,37 +589,6 @@ template <typename T> class CMedianFilter
 };
 
 bool NewThread(void(*pfn)(void*), void* parg);
-
-#ifdef WIN32
-inline void SetThreadPriority(int nPriority)
-{
-    SetThreadPriority(GetCurrentThread(), nPriority);
-}
-#else
-
-#define THREAD_PRIORITY_LOWEST          PRIO_MAX
-#define THREAD_PRIORITY_BELOW_NORMAL    2
-#define THREAD_PRIORITY_NORMAL          0
-#define THREAD_PRIORITY_ABOVE_NORMAL    -2
-#define THREAD_PRIORITY_HIGHEST        PRIO_MIN
-
-inline void SetThreadPriority(int nPriority)
-{
-    // It's unclear if it's even possible to change thread priorities on Linux,
-    // but we really and truly need it for the generation threads.
```

Comment:
It is only possible to decrease priority (increase nice) which is kind of sad. Remove it then.